docs/_layouts/character.html

```diff
@@ -41,9 +41,30 @@
             <div class="col-sm-3 col-md-2 sidebar">
                 {% include sidebar-nav.html character=page.character style=page.style %}
             </div>
-            <!-- <div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
+            
+            <!-- defines thisPage using the data file characters.json for ease -->
+            {% for aCharacter in site.data.characters %}
+                    {% if page.character == aCharacter.name and page.style == aCharacter.style %}
+                        {% assign thisPage = aCharacter %}
+                    {% endif %}
+                    {% if thisPage.awards.name %}
+                        {% assign awardsFirst = "/images/" | append: thisPage.awards.first | append: "_w.png" %}
+                        {% assign awardsSecond = "/images/" | append: thisPage.awards.second | append: "_w.png" %}
+                        {% assign awardsThird = "/images/" | append: thisPage.awards.third | append: "_w.png" %}
+                    {% endif%}
+                    {% if page.character == aCharacter.awards.name and page.style == aCharacter.awards.style %}
+                        {% assign feeder = aCharacter %}
+                        {% assign feederFirst = "/images/" | append: feeder.awards.first | append: "_w.png" %}
+                        {% assign feederSecond = "/images/" | append: feeder.awards.second | append: "_w.png" %}
+                        {% assign feederThird = "/images/" | append: feeder.awards.third | append: "_w.png" %}
+                    {% endif %}
+            {% endfor %}
+            {% assign firstColor = "/images/" | append: thisPage.colors.first | append: "_w.png"%}

```

Comment:
This should be replaced with an include similar to the AP FA glyph, to include the color wedge.